src/WinRT.Runtime/IWinRTObject.net5.cs

```diff
@@ -35,7 +35,7 @@ bool IDynamicInterfaceCastable.IsInterfaceImplemented(RuntimeTypeHandle interfac
             return IsInterfaceImplementedFallback(interfaceType, throwIfNotImplemented);
         }
 
-        bool IsInterfaceImplementedFallback(RuntimeTypeHandle interfaceType, bool throwIfNotImplemented)
+        internal sealed bool IsInterfaceImplementedFallback(RuntimeTypeHandle interfaceType, bool throwIfNotImplemented)

```

Comment:
I think these fallbacks can be called by RCW types. Can we make them `protected` or `protected internal` instead?

```suggestion
        protected sealed bool IsInterfaceImplementedFallback(RuntimeTypeHandle interfaceType, bool throwIfNotImplemented)

```

Reply:
> "can be called by RCW types"

Just so I understand, do you have some scenario in mind? The generated RCWs will never call those methods (and there's no references to them at all in `code_writers`, which also how the CI still passed. Can you think of a case where a user would want/need to actually manually implement an RCW type and then call those fallback methods? I can't personally think of one. And would that even be a scenario we want/need to support in the first place? 🤔